Kwenta/kwenta
# Increase the max total rows of the tables and add pagination

## Description
To meet the new requirements of the escrow table, we made the following changes:
1. Increase the maximum number of total rows from 1000 to 9999.
2. Add pagination

## Related issue


## Motivation and Context
Show all the rows of escrow entries

## How Has This Been Tested?
1. Open a watch-only wallet with kethic.eth
2. Scroll down to the bottom
3. See the pagination (11 pages) correctly 

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/207138641-9a8051e3-bb9a-4ea8-8a48-bfaf0cd9fe48.png)
